assets/charts/control-plane/calico/values.yaml

```diff
@@ -13,3 +13,5 @@ calico:
   podCIDR: 10.2.0.0/16
   networkEncapsulation: "ipipMode: Always"
   blockedMetadataCIDRs: []
+  # Lokomotive specific change.
+  failsafeInboundHostPorts: []
```

Comment:
Shouldn't this here be `~` (null) if we always render it in `failsafeInboundHostPorts: {{ .Values.calico.failsafeInboundHostPorts }}` ?

As we do not override it in `assets/terraform-modules/bootkube/resources/charts/calico.yaml` if value is `null`, it will still be set to `[]`.

Reply:
Except for Packet, with this implementation if the value is `null` nothing will be applied, because only 2 conditions are checked `%{~ if failsafe_inbound_host_ports != null && failsafe_inbound_host_ports != [] ~}` and `%{~ if failsafe_inbound_host_ports == [] ~}1

Reply:
Yes, but this file (`assets/charts/control-plane/calico/values.yaml`) defines default values for the chart, which won't be overwritten. That means the following command should not return anything:
```sh
$ helm template calico ./assets/charts/control-plane/calico | grep failsafeInboundHostPorts
  failsafeInboundHostPorts: []
```

However, it currently does.

Reply:
This is for packet right?

Reply:
aah, I got it what you mean

Reply:
yes, this should be null